File: furdega/Dtos/HomePage/Output/WorkExampleResponse.cs
namespace Furdega.Dtos.HomePage.Output
{
    public class WorkExampleResponse
    {
        public string Title { get; set; }

        public string FurnitureType { get; set; }
        public string Duration { get; set; }
        public string WorkType { get; set; }

        public string Description { get; set; }

        public ImageResponse BeforeImage1 { get; set; }
        public ImageResponse BeforeImage2 { get; set; }
        public ImageResponse BeforeImage3 { get; set; }

        public ImageResponse AfterImage1 { get; set; }
        public ImageResponse AfterImage2 { get; set; }
        public ImageResponse AfterImage3 { get; set; }
    }
}

File: furdega/Frontend/src/types/work-examples-section/work-example-response.ts
import { ImageResponse } from "../image-response"

export type WorkExampleResponse = {
  title: string
  furnitureType: string
  duration: string
  workType: string
  description: string
  beforeImage1: ImageResponse
  beforeImage2: ImageResponse
  beforeImage3: ImageResponse
  afterImage1: ImageResponse
  afterImage2: ImageResponse
  afterImage3: ImageResponse
}
